### PR TITLE
package_info readme file dependency constraints corrected.

### PR DESCRIPTION
**Pull request by Roshan13046**
## Description

package_info readme file was wrong: dependency constraint single quote. 
corrected dependency constraint with double quotes.

## Related Issues

flutter/flutter#53413

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Review by cyanglaz — COMMENTED** [review]
Thank you for the PR, change LGTM.
Could you please also bump the patch version of the plugin, as well as add an entry in the CHANGLOG.md?

**Comment by Roshan13046** [comment]
> add an entry in the CHANGLOG.md?

Yes, I will make the above changes.
To make the above changes do I have to revert to my current PR?

File: packages/package_info/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.4.3+4
+
+* Update package_info readme file dependency constraints corrected.
```
Comment (cyanglaz) [inline packages/package_info/CHANGELOG.md:3]: How about:
```
Update README: fix dependency constraint syntax.
```

**Comment by Roshan13046** [comment]
Please check my PR.

**Comment by cyanglaz** [comment]
Looking at the diff again. What is wrong with the single quote syntax? I don't think there's any difference between ' and ". 
Some official docs also uses single quote. https://flutter.dev/docs/development/packages-and-plugins/using-packages#conflict-resolution

**Comment by stuartmorgan** [comment]
Both `'` and `"` are supported. We tend to use `"`, but revving a bunch of plugins (there are 15+ copies of this in the repo at this point) for a style choice (especially of a temporary message) isn't worth the overhead it would entail. Sorry we didn't catch this on initial review.

**Pull request closed without merging**